AcadeMeData/tests/test_client.py

```diff
@@ -11,3 +11,9 @@ def test_homepage(client):
 def test_contactus(client):
     response = client.get('/contact_us/')
     assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_course_list(client):
+    response = client.get('/course-list/')
+    assert response.status_code == 200
```

Comment:
You could test the context here
that is needed by the template
  context['seluniversity'] = 
    context['seldegree'] =

Reply:
will do!

Reply:
@Yarboa 
Done. The asked test can be found on `AcadeMeData/tests/test_views.py`